File: examples/runtime/runtime-error.c
#include <stdio.h>

float foo(int a, int b)
{
    return a / b;
}

float bar(int a, int b)
{
    return foo(a, b);
}

int main(int argc, char* argv[])
{
    if(argc != 3)
    {
        printf("USAGE: runtime-error.py NUM1 NUM2");
        return 1;
    }

    int a,b,x;

    sscanf(argv[1], "%i", &a);
    sscanf(argv[2], "%i", &b);

    x = bar(a, b);

    printf("a / b = %i\n", x);

    return 0;
}

File: examples/runtime/runtime-error.py
#!/usr/bin/python3

import sys

def foo(a, b):
    return a // b

def bar(a, b):
    return foo(a, b)

if __name__ == "__main__":
    if len(sys.argv) != 3:
        print("USAGE: runtime-error.py NUM1 NUM2")
        sys.exit(1)

    a = int(sys.argv[1])
    b = int(sys.argv[2])

    x = bar(a, b)

    print("a / b = {}".format(x))